Dockerfile.test

```diff
@@ -0,0 +1,13 @@
+# docker build . -f Dockerfile.test -t test_builder
+
+FROM ubuntu
+
+RUN apt update -y && apt install -y cmake gcc g++ make
+COPY . /tmp
+RUN cd /tmp && \
+    mkdir build && \
+    cd build && \
+    cmake .. -DLIBMBUS_BUILD_EXAMPLES=ON -DLIBMBUS_BUILD_TESTS=ON -DLIBMBUS_ENABLE_COVERAGE=ON && \
+    cmake --build . -j && \
+    cd .. && \
+    ./test/generate-xml.sh test/test-frames
```

Comment:
This change looks unrelated to the pull request description. Why is it not mentioned there?

Reply:
I'm quite new to github.
So the text in the PR is not that good of a description.

This commit might fit better in the other PR (#165), it was useful for me during development, to see that the tests are actually ok.

Reply:
Okay, please move it.